src/main/msp/msp.c

```diff
@@ -2900,9 +2902,10 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         break;
 
     case MSP_SET_GPS_RESCUE_PIDS:
-        gpsRescueConfigMutable()->throttleP = sbufReadU16(src);
-        gpsRescueConfigMutable()->throttleI = sbufReadU16(src);
-        gpsRescueConfigMutable()->throttleD = sbufReadU16(src);
+        positionControlConfigMutable()->altitude_P = sbufReadU16(src);
+        positionControlConfigMutable()->altitude_I = sbufReadU16(src);
+        positionControlConfigMutable()->altitude_D = sbufReadU16(src);
+        // altitude_F not implemented yet
```

Comment:
Does this comment mark a todo that has been forgotten?

Reply:
not really,  I didn't know if it should, or should not, be included at this point.
I figure it should, but I am never comfortable about adding things to MSP, and wasn't sure exactly how to do that.
It's not a particularly essential thing to tune, so I figured it could wait, but if you explained exactly what to do, so I didn't break it, I'll do it.